bin/determineKnnData.js

```diff
@@ -17,7 +17,24 @@ body_image_files.forEach(fileName => {
         numPixels++;
       }
     }
-    console.log(image.src, (1.0 * image.width) / image.height, numPixels);
+    const name = fileName.split('.')[0];
+    const src = `${name}_image`;
+
+    console.log(`import ${src} from '../../${imagePath}'`);
```

Comment:
Intentional logging?

Reply:
Yep! This script outputs a bunch of lines to put into `fishData`. It can probably be improved to put them in a more usable order though...

Reply:
Yea I was just using `grep` to work around that for now, but it would definitely be an improvement.

Reply:
oh that's smart. I just ran it twice with various lines commented out. It was still super helpful and probably saved me an hour or so of writing boilerplate!